src/main/java/nl/tudelft/semgroup4/GameEndedState.java

```diff
@@ -22,208 +16,392 @@
+                textFieldPlayer2.getHeight());
+
+    }
+
+    /**
+     * Initializes the textFields.
+     * 
+     * @param container
+     *            container in which the textFields are initialized
+     */
+    private void initializeTextFields(GameContainer container) {
+        textFieldPlayer1 = new TextField(container, typeFontPlayer,
+                container.getWidth() / generalWidth * 4,
+                container.getHeight() / 3, 200, 50);
+        textFieldPlayer1.setConsumeEvents(true);
+        textFieldPlayer1.setMaxLength(10);
+
+        textFieldPlayer2 = new TextField(container, typeFontPlayer,
+                container.getWidth() / generalWidth * 4,
+                container.getHeight() / 2, 200, 50);
+        textFieldPlayer2.setConsumeEvents(true);
+        textFieldPlayer2.setMaxLength(10);
+    }
+
+    /**
+     * Draws the information of both players on the screen.
+     * 
+     * @param container
+     *            container in which to draw onto.
+     */
+    private void drawPlayersInfo(GameContainer container, Graphics graphics) {
+        for (int i = 0; i < players.length; i++) {
+            typeFontPlayer.drawString(container.getWidth() / 8,
+                    container.getHeight() / (3 - i), "PLAYER " + (i + 1),
+                    Color.white);
+            typeFontPlayer.drawString(container.getWidth() / generalWidth * 2,
+                    container.getHeight() / (3 - i),
+                    Integer.toString(players[i].getScore()), Color.white);
+            typeFontPlayer.drawString(container.getWidth() / generalWidth * 3,
+                    container.getHeight() / (3 - i),
+                    Integer.toString(players[i].getMoney()), Color.white);
+
+            graphics.drawRect(container.getWidth() / generalWidth * 4,
+                    container.getHeight() / (3 - i),
+                    textFieldPlayer1.getWidth() - 1,
+                    textFieldPlayer1.getHeight() - 1);
+        }
+
+        textFieldPlayer1.render(container, graphics);
+        textFieldPlayer2.render(container, graphics);
+    }
+
+    /**
+     * Draws the title according to of the game has been won or lost.
+     * 
+     * @param container
+     *            container in which to draw the title
+     */
+    private void drawTitle(GameContainer container) {
+        if (won) {
+            gameTitleText = "GAME COMPLETED";
+            typeFontTitle.drawString(
+                    container.getWidth() / 2
+                            - typeFontTitle.getWidth(gameTitleText) / 2,
+                    container.getHeight() / 8, gameTitleText, Color.yellow);
+        } else {
+            gameTitleText = "GAME LOST";
+            typeFontTitle.drawString(
+                    container.getWidth() / 2
+                            - typeFontTitle.getWidth(gameTitleText) / 2,
+                    container.getHeight() / 8, gameTitleText, Color.yellow);
+        }
+
```

Comment:
If you take the drawString out of the if else statement and just put it after it, you only have to call it once. Makes the code look neater.